presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushAggregationThroughOuterJoin.java

```diff
@@ -272,7 +272,7 @@ private MappedAggregationInfo createAggregationOverNull(AggregationNode referenc
             Symbol aggregationSymbol = entry.getKey();
             AggregationNode.Aggregation aggregation = entry.getValue();
             AggregationNode.Aggregation overNullAggregation = new AggregationNode.Aggregation(
-                    (FunctionCall) new ExpressionSymbolInliner(sourcesSymbolMapping).rewrite(aggregation.getCall()),
+                    (FunctionCall) new ExpressionSymbolInliner(sourcesSymbolMapping::get).rewrite(aggregation.getCall()),
```

Comment:
👍  and then you could even combine instantiation with call to `rewrite`. No longer "factory method" but perhaps even better